bodhi/server/models.py

```diff
@@ -25,16 +25,8 @@
 from textwrap import wrap
 from datetime import datetime
 from collections import defaultdict
-
-try:
-    # python3
-    from urllib.parse import quote
-except ImportError:
-    from urllib import quote
```

Comment:
PEP-8 states that imports should be grouped into three groups, at the top of the file:

1. Standard library
2. Third party
3. Your package

Since urlib is from the standard library, it should stay here next to other standard library imports (like datetime and collections).